### PR TITLE
add third party jython lib

### PR DESCRIPTION
**Pull request by OscarQQ**
related: https://github.com/eclipse/openj9/issues/12284
Signed-off-by: Yixin Qian <Yixin.Qian@ibm.com>

### CODE REVIEW

**Comment by OscarQQ** [comment]
test: internal Grinder_TKG/1180/
@renfeiw @llxia could you take a review? thanks

**Review by llxia — APPROVED** [review]
LGTM

File: scripts/getDependencies.pl [hunk 1]
```diff
@@ -128,6 +128,11 @@
 		url => ' https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.osgi/3.16.100/org.eclipse.osgi-3.16.100.jar',
 		fname => 'org.eclipse.osgi-3.16.100.jar',
 		sha1 => '7ddb312f386b799d6e004d193a01c50169bf69f3'
+	},
+	jython => {
+		url => ' https://repo1.maven.org/maven2/org/python/jython-standalone/2.7.2/jython-standalone-2.7.2.jar',
+		fname => 'jython-standalone-2.7.2.jar',
```
Comment (llxia) [inline scripts/getDependencies.pl:134]: Could we remove the version in the name? In this way, if we upgrade the version, we do not need to update the related tests in other repos.

**Review by renfeiw — APPROVED** [review]
Apart from Lan's comment, everything else looks good to me.

**Review by llxia — CHANGES_REQUESTED** [review]
Requested the change for jar name

**Review by llxia — APPROVED** [review]
LGTM

**Comment by llxia** [comment]
https://ci.eclipse.org/openj9/job/test.getDependency/12710/

**Pull request merged**